lms/envs/test.py

```diff
@@ -26,6 +26,11 @@
 
 from openedx.core.lib.tempdir import mkdtemp_clean
 
+# This patch disabes the commit_on_success decorator during tests
+# in TestCase subclasses.
+from util.testing import patch_testcase
```

Comment:
@singingwolfboy `util` is an app in `common/djangoapps` and this is how it is being referenced everywhere in the codebase.